File: API/Models/Venda.cs
namespace API.Models
{
    public class Venda
    {
        public int Id { get; set; }
        public string Cliente { get; set; }
        public ItemVenda ItemVenda { get; set; }
        public int ItemVendaId { get; set; }
        public int FormaId { get; set; }
        public FormaPagamento FormaPagamento { get; set; }
    }
}

File: APP/src/app/models/venda.ts
import { ItemVenda } from './item-venda';
import { FormasPag } from './formasPag';


export interface Venda{
    id?:number;
    cliente: string;
    formaId: number;
    formaPag?: FormasPag[];
    itemVendaId: number;
    itemVenda?: ItemVenda[];
}